### PR TITLE
refactor to avoid use of globally defined variables

### LINKED ISSUE #11: Refactor to avoid use of global variables
## Current Behaviour
The config variables are defined at the top of the python file and all functions can make use of them.

## Changes Requested
Edit code so only the relevant configs are passed into each function as an argument.

### PR DESCRIPTION
**Pull request by funbeedev**
### 2. Description
Removed use of global variables. 
Config file reading performed in a function. 
Variables are now passed into each function as needed.

### 3. Fill in checklist by marking [x]

- [X] I've read the CONTRIBUTING.md
- [X] I was assigned to this issue
- [X] My code is formatted using required linting
- [X] I've run my code locally and checked it works

### CODE REVIEW

File: automate_git.py [hunk 1]
```diff
@@ -140,21 +146,23 @@ def start_program_flow():
 
     print("------ Start ------")
 
-    create_local_git_repo()
+    access_token, repo_name, first_file, msg, github_name, pc_directory, repo_directory, run_cmd = setup_config()
```
Comment (Stedders) [inline automate_git.py:149]: Nice :+1:

File: automate_git.py [hunk 2]
```diff
@@ -4,23 +4,30 @@
...
-config = ConfigParser()
-config.read(config_file)
 
-# get all settings from config file
-access_token = config.get('your_settings', 'github_token')
-repo_name = config.get('your_settings', 'repo_name')
-first_file = config.get('your_settings', 'commit_file')
-msg = config.get('your_settings', 'first_commit_msg')
-github_name = config.get('your_settings', 'github_name')
+def setup_config():
```
Comment (Stedders) [inline automate_git.py:8]: @funbeedev  - how about passing the config file in to the function as well...

```python
def setup_config(config_file = None):
    if config_file is None:
        config_file = 'config.ini'
```

Default to `None` is the standard way of [defaulting parameters](https://docs.python.org/3/tutorial/controlflow.html#default-argument-values).

**Comment by Stedders** [comment]
That looks good, my one small suggestions would be to make the file configurable and default to your standard value.

**Comment by funbeedev** [comment]
@Stedders Thanks for the review! I'll be making the file configurable in a separate PR 👍🏿

**Pull request merged**